### PR TITLE
Fix RESP detection

### PR DESCRIPTION
**Pull request by 0xA50C1A1**
Please sign (check) the below before submitting the Pull Request:

- [x] I have signed the ntop Contributor License Agreement at https://github.com/ntop/legal/blob/main/individual-contributor-licence-agreement.md
- [x] I have read the contributing guide lines at https://github.com/ntop/nDPI/blob/dev/CONTRIBUTING.md
- [x] I have updated the documentation (in doc/) to reflect the changes made (if applicable)

Link to the related issue: #2287

Describe changes:

Fixed broken dissector, added RESP pcap sample and renamed the protocol ID from NDPI_PROTOCOL_REDIS to NDPI_PROTOCOL_RESP for correctness.

### CODE REVIEW

File: src/lib/protocols/resp.c [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    if (!isascii(packet->payload[offset]))
+      goto exclude;
+    offset++;
+  }
+
+  if (memcmp(&packet->payload[offset], "\r\n", 2) == 0) {
+    NDPI_LOG_INFO(ndpi_struct, "found RESP\n");
+    ndpi_set_detected_protocol(ndpi_struct, flow, NDPI_PROTOCOL_RESP,
+                               NDPI_PROTOCOL_UNKNOWN, NDPI_CONFIDENCE_DPI);
+  }
```
Comment (0xA50C1A1) [inline src/lib/protocols/resp.c:66]: @IvanNardi @utoni what do you think about that? Could it be optimized in some way?

Comment (IvanNardi) [reply src/lib/protocols/resp.c:66]: `ndpi_isdigit` instead of `isascii`? [if we are looking for a number...]

**Review by IvanNardi — CHANGES_REQUESTED** [review]
I think you did something like
```
git rm src/lib/protocols/redis_net.c
git add src/lib/protocols/resp.c
```
Could you try reverting and doing
```
git move src/lib/protocols/redis_net.c src/lib/protocols/resp.c
```
?
I prefer to keep the file history

**Comment by 0xA50C1A1** [comment]
> I think you did something like
> 
> ```
> git rm src/lib/protocols/redis_net.c
> git add src/lib/protocols/resp.c
> ```
> 
> Could you try reverting and doing
> 
> ```
> git move src/lib/protocols/redis_net.c src/lib/protocols/resp.c
> ```
> 
> ? I prefer to keep the file history

I've tried. For some reason redis_net.c is still considered as deleted.

**Pull request closed without merging**